.env.alfajores

```diff
@@ -55,7 +55,7 @@ AZURE_ORACLE_CENTRALUS_FULL_NODES_WS_PORT="8546"
 
 # Temporarily point to celo-org repository to consume patched image.
 GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-org/geth"
-GETH_NODE_DOCKER_IMAGE_TAG="1.3.2"
+GETH_NODE_DOCKER_IMAGE_TAG="1.5.0-beta.1"
```

Comment:
This change is fine, but we will be releasing another version that has the espresso HF activation number for alfajores baked in (it's currently not specified).

Reply:
thanks for the heads up, will upgrade that again later then!